vscode/microsoft-kiota/src/utilities/deep-linking.ts

```diff
@@ -103,14 +109,38 @@ export function validateDeepLinkQueryParams(queryParameters: Partial<Integration
     errormsg.push("Invalid parameter 'type' deeplinked. Expected values: " + acceptedPluginTypes.join(","));
   }
 
+  if (projectPath && !path.isAbsolute(projectPath)) {
+    projectPath = undefined;
+    errormsg.push(`A relative paths is not supported for the projectPath parameter`);
+  }
+
   validQueryParams = {
-    descriptionurl: descriptionurl,
-    name: name,
+    descriptionurl,
+    name,
     kind: validKind,
     type: providedType,
     language: givenLanguage,
-    source: source,
-    ttkContext: queryParameters.ttkContext ? queryParameters.ttkContext : undefined
+    source,
+    ttkContext: queryParameters.ttkContext ? queryParameters.ttkContext : undefined,
```

Comment:
I don't think the check is necessary since we're setting ttkContext to undefined anyway.
```suggestion
    ttkContext: queryParameters.ttkContext,
```